Tools/autotest/apmrover2.py

```diff
@@ -23,7 +24,7 @@ def arm_rover(mavproxy, mav):
     wait_seconds(mav, 30)
 
     mavproxy.send('arm throttle\n')
-    mavproxy.expect('ARMED')
+    mavproxy.expect(u'ARMED')
```

Comment:
From what I understand, python 3 string are unicode by default, python 2 are bytes. The problem is that pexpect doesn't work with byte string. I take some time to find this solution and use spawnu instead of spawn for pexpect. Another solution could be to use from __future__ import unicode_literals that make all string unicode witout 'u' . But I need to check if it works both in python 2 and 3 and with all string that are not supposed to be unicode.